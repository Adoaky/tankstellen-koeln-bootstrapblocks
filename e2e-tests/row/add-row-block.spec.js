/**
 * WordPress dependencies
 */
import {
	createNewPost,
	searchForBlock,
	getEditedPostContent,
} from '@wordpress/e2e-test-utils';

describe( 'adding row block', () => {
	beforeEach( async () => {
		await createNewPost();
	} );

	it( 'Should insert row block with the global inserter', async () => {
		await searchForBlock( 'Bootstrap Row' )
		await page.click( 'button.editor-block-list-item-wp-bootstrap-blocks-row' )

		expect( await page.$( '[data-type="wp-bootstrap-blocks/row"]' ) ).not.toBeNull();
		expect( await page.$$( '[data-type="wp-bootstrap-blocks/column"]' ) ).toHaveLength(2);

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );
