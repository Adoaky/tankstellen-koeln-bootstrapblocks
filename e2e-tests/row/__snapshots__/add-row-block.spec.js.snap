// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adding row block Should insert row block with the global inserter 1`] = `
"<!-- wp:wp-bootstrap-blocks/row -->
<!-- wp:wp-bootstrap-blocks/column {\\"sizeMd\\":6} /-->

<!-- wp:wp-bootstrap-blocks/column {\\"sizeMd\\":6} /-->
<!-- /wp:wp-bootstrap-blocks/row -->"
`;
